home/src/main/resources/rdf/i18n/sr_Latn_RS/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -6309,3 +6309,52 @@ uil-data:captcha_user_sol_invalid.Vitro
         uil:hasApp      "Vitro" ;
         uil:hasKey      "captcha_user_sol_invalid" ;
         uil:hasPackage  "Vitro-languages" .
+
+uil-data:suppress_operation_for_roles.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Zabrani <b>{0}</b> operacije za uloge"@sr-Latn-RS ;
```

Comment:
```suggestion
        rdfs:label       "Zabrani <b>{0}</b> operacije za role"@sr-Latn-RS ;
```
@chenejac I'm not sure about this, it is used here as a slang of some sorts, I don't know if it's an official word :smile: "uloge" just doesn't sound good to me